package/Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN make -C /helm
 FROM registry.suse.com/bci/bci-base:15.5.36.5.20 AS build
 ARG ARCH=amd64
 RUN zypper -n install curl gzip tar
-ENV KUBECTL_VERSION v1.24.16
+ENV KUBECTL_VERSION v1.26.9
 ENV K9S_VERSION=v0.27.4
 ENV KUSTOMIZE_VERSION=v5.1.1
```

Comment:
I believe this is fine, but this version of Kustomize is higher than the one that ships with kubectl `v1.26.9`. in the future we should probably stop installing Kustomize directly and just add an alias to `kubectl kustomize`